### PR TITLE
fix(cmd): Drop cobra subcommands

### PR DESCRIPTION
**Pull request by jotadrilo**
This patch drops the `sync` and `syncChart` cobra subcommands.

### CODE REVIEW

**Comment by jotadrilo** [comment]
Tests expect to sync a few charts only. We need to support syncing a list of charts (#42) only first.

**Review by tompizmor — COMMENTED** [review]
the changes look good, however, what I don't like is that if you just execute the binary it will start syncing charts (if it can find a proper config file)

I would expect that calling the binary without arguments is the same as calling it with `--help` flag.

**Comment by jotadrilo** [comment]
> the changes look good, however, what I don't like is that if you just execute the binary it will start syncing charts (if it can find a proper config file)
> 
> I would expect that calling the binary without arguments is the same as calling it with `--help` flag.

What if we make the `--config` flag a mandatory flag? This way the binary will sync charts only if you provide a configuration file to read the source/target repos from.

**Comment by jotadrilo** [comment]
I am also OK with keeping Cobra with a single command. Whatever you prefer.

**Comment by tompizmor** [comment]
I prefer to keep one Cobra command, also because in the help output you see a clear distinction between command and flag implemented by the developer of the tool (`config`, `dry-run`, `from-date`) and the regular flags added by default (`log_dir`, `log_file`, etc)

File: cmd/sync.go [hunk 1]
```diff
@@ -1,96 +1,89 @@
...
+	klog.Infof("Looking for the default config %s", defaultCfgFile)
+	return errors.Trace(viper.ReadInConfig())
+}
+
+func newSyncCmd() *cobra.Command {
+	var c api.Config
+
+	cmd := &cobra.Command{
+		Use:     "sync",
+		Short:   "Syncronizes two chart repositories",
```
Comment (tompizmor) [inline cmd/sync.go:56]: `Syncronize` should be `Synchronize`. I see these typos in other parts too, can you do a quick search and replace??

File: cmd/root.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+	cmd.PersistentFlags().StringVarP(&rootConfig, "config", "c", "", fmt.Sprintf("Config file. Defaults to ./%s or $HOME/%s)", defaultCfgFile, defaultCfgFile))
+
+	// Add subcommands
+	cmd.AddCommand(
+		newSyncCmd(),
+	)
+
+	// Workaround to disable help subcommand
+	// https://github.com/spf13/cobra/issues/587
+	cmd.SetHelpCommand(&cobra.Command{
```
[the remark this answers is not in the document]
Comment (jotadrilo) [reply cmd/root.go:41]: Exactly, since we can use `chartsyncer -h` or `chartsyncer --help`

**Review by tompizmor — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**